src/components/pages/MatchUp/MatchUpContainer.js

```diff
@@ -39,20 +35,10 @@ function MatchUpContainer({ LoadingComponent, ...props }) {
 
   useEffect(() => {
     getChildSquad(authState, props.child.id).then(squad => {
-      getFaceoffsForMatchup(authState, squad.ID).then(allFaceoffs => {
```

Comment:
I dont thin this was necessary to delete, I see what you're doing, making our repo reflect what Team E (I believe) has in their repo. But i dont think it was necessary, if you looked at where the getFaceoffsForMatchup function was made you'd see that it is the same as getChildFaceoffs, save for 1 parameter that was needed for emoji feedback functionality. Is there a specific reason why you deleted these things, or did you just want to make our repo look exactly like Team E's repo?

Reply:
Ah yes, for getChildFaceoffs, I just put that in because it was in Team D's repo along with the locking mechanism code, I believe...  But I could change it back, along with the first deletion.  Let's zoom though. I'd like to look over the code with you.

Reply:
Oh ok yea, we can figure out if we're meant to make our repo look exactly like theirs or more so merge their code into our. I also think that if ours works it works and we dont need to do extra work by making everything exactly the same.